docs/release-notes/index.md

```diff
@@ -0,0 +1,1243 @@
+---
+navigation_title: "Elastic APM Java Agent"
+mapped_pages:
+  - https://www.elastic.co/guide/en/apm/agent/java/current/release-notes-1.x.html
+  - https://www.elastic.co/guide/en/apm/agent/java/current/release-notes.html
+---
+
+# Elastic APM Java Agent release notes [elastic-apm-java-agent-release-notes]
+
+Review the changes, fixes, and more in each version of Elastic APM Java Agent.
+
+To check for security updates, go to [Security announcements for the Elastic stack](https://discuss.elastic.co/c/announcements/security-announcements/31).
+
```

Comment:
I just noticed that everything which previously was under a `Potentially Breaking Changes` section is missing in the new changelog, e.g. [like this entry](https://www.elastic.co/guide/en/apm/agent/java/current/release-notes-1.x.html#_potentially_breaking_changes).
Is this intentional or due to some automation?

Reply:
It looks like those were removed in https://github.com/elastic/asciidocalypse/pull/59/files#diff-092dcad3448387f6eaed42f5ee8c4785c596589c4cf1a57dff40c654458f6513L106, but were not moved to a `breaking-changes.md` file. @KOTungseth was this intentional? Should we create a breaking changes page? Is that something you could help with?